lib/__tests__/is-disabled.ts

```diff
@@ -133,6 +175,6 @@ describe('assert.dom(...).isDisabled()', () => {
     expect(() => assert.dom(undefined).isDisabled()).toThrow('Unexpected Parameter: undefined');
     expect(() => assert.dom({}).isDisabled()).toThrow('Unexpected Parameter: [object Object]');
     expect(() => assert.dom(document).isDisabled()).toThrow('Unexpected Parameter: [object Document]');
-    expect(() => assert.dom(document.createElement('div')).isDisabled()).toThrow('Unexpected Element Type: [object HTMLDivElement]');
+    expect(() => assert.dom(document.createElement('div')).isDisabled()).toThrow('Generic Element Type: [object HTMLDivElement] does not use aria-disabled attribut');
```

Comment:
May I suggest instead of "does not use" maybe try "should use" instead? It seems more consistent with [RFC 2119](https://tools.ietf.org/html/rfc2119) style of language. I know this isn't an RFC but the consistency feels nice.

Reply:
👍 a bit late here, but thanks will add the change